starter-code/routes/celebrities.routes.js

```diff
@@ -0,0 +1,90 @@
+const express = require('express')
+const router = express.Router()
+const CelebrityModel = require('../models/celebrity')
+
+
+router.get('/celebrities', (req, res, next) => {
+    CelebrityModel.find()
+        .then((celebrity)=>{
+            // console.log(celebrity)
+            res.render('./celebrities/index.hbs', {celebrity})
+        })
+        .catch((err)=>{
+           console.log('Error is:', err)
+           next()
+        })
+})
+
+
+router.get('/celebrities/new', (req, res, next)=>{
+    res.render('./celebrities/new.hbs')
+})
+
+router.get('/celebrities/:id/edit',(req, res, next)=>{
+    CelebrityModel.findById(req.params.id)
+    .then((celeb)=>{
+        console.log('Data added')

```

Comment:
you are not adding any data here. This route is just fetching information from the DB to render it in the view.